ESP32/main/src/wifi/WifiController.cpp

```diff
@@ -0,0 +1,213 @@
+#include "../../config.h"
+#ifdef IS_WIFI
+#include "WifiController.h"
+namespace WifiController
+{
+
+    const String mSSIDAP = F("UC2");
```

Comment:
Can we also add this parameter to the Json document somehow?

Reply:
What means `F(*)` ;-)

Reply:
im using it as fallback, if connecting to a network failed, it reopen uc2 ap

F(*) is like PRAGMA. string get stored in flash mem instead of sram